go/cmd/vttablet/vttablet.go

```diff
@@ -43,12 +43,17 @@ func main() {
 	dbconfigs.RegisterFlags(dbconfigFlags)
 	mysqlctl.RegisterFlags()
 	flag.Parse()
-	tabletenv.Init()
 	if len(flag.Args()) > 0 {
 		flag.Usage()
 		log.Errorf("vttablet doesn't take any positional arguments")
 		exit.Return(1)
 	}
+	if err := tabletenv.VerifyConfig(); err != nil {
+		log.Errorf("invalid config: %v", err)
+		exit.Return(1)
```

Comment:
The previous code didn't use `log.Fatalf` because our custom wrapper `exit.Return()` was used to exit the main Go routine.

I don't think that's necessary and a regular `os.Exit()`, called via `log.Exit()` is good enough. I've changed it accordingly.